package.json

```diff
@@ -23,6 +23,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.6.0",
+    "react-number-format": "^5.1.1",
```

Comment:
i am thinking to use https://mikemcl.github.io/bignumber.js/#toFor to format number display in metascan, since we already using `BigNumber`. What do you think?

it supports:
- suffix
- prefix
- thousand separator

Reply:
Good Idea, Will remove this and will use BigNumber